File: C/PRAK603-2210817120009-Najah_Maisyaroh.c
#include <stdio.h>
int main(){
    int baris1, baris2, i, hasil;
    scanf("%d %d", &baris1, &baris2);
    if(baris1!=baris2){
        printf("Jumlah tidak sama");
        return 0;}
    int a[baris1];
    int b[baris2];
    for(i=0;i<baris1;i++){
        scanf("%d", &a[i]);}
    for(i=0;i<baris2;i++){
        scanf("%d", &b[i]);}
    for(i=0; i<baris1; i++){
        hasil=a[i]*b[i];
        printf("%d ", hasil);}
    return 0;
}

File: Python/PRAK603-2210817120009-Najah_Maisyaroh.py
batas1, batas2=map(int,input().split())
if(batas1!=batas2):
	print("Jumlah tidak sama")
else:
	matrix1=list(map(int, input().split()))
	matrix2=list(map(int, input().split()))
	for i in range(batas1):
		print(matrix1[i]*matrix2[i], end=" ")
	print()